geoportailv3/static/js/maincontroller.js

```diff
@@ -127,6 +127,11 @@ app.MainController = function($scope, ngeoGetBrowserLanguage, gettextCatalog,
   this['userOpen'] = false;
 
   /**
+   * @type {string|undefined}
+   */
+  this['infosAppSelector'] = undefined;
```

Comment:
it keeps the information if there is a location info window being shown, or a layer information window being shown , or nothing. I think with strings, this is the clearest way